libr/bin/format/pe/pemixed.c

```diff
@@ -0,0 +1,174 @@
+#include <stdio.h>
+#include "pemixed.h" 
+
+static bool check_il_only(ut32 flags);
+
+static int r_bin_pemixed_init(struct r_bin_pemixed_obj_t* bin, struct PE_(r_bin_pe_obj_t)* pe_bin){
+    struct PE_(r_bin_pe_obj_t)* sub_bin_dos;
+    struct PE_(r_bin_pe_obj_t)* sub_bin_native;
+    struct PE_(r_bin_pe_obj_t)* sub_bin_net;
+
+    sub_bin_dos = r_bin_pemixed_init_dos(pe_bin);
```

Comment:
follow the r2 coding style please.

* add a space before every ( that is not a function signature
* add a space before { too

Reply:
also, we use tabs to indent code, not spaces